components/checkbox/style/index.ts

```diff
@@ -36,6 +36,13 @@ export const genCheckboxStyle: GenerateStyle<CheckboxToken> = (token) => {
         ...resetComponent(token),
 
         display: 'inline-flex',
+        flexWrap: 'wrap',
+        columnGap: '8px',
+
+        // Group > Grid
+        '.ant-row': {
+          width: '100%',
+        },
```

Comment:
```suggestion
        [`${token.antCls}-row`]: {
          flex: 1,
        },
```
1. `ant` 前缀要用 token，这是动态的
2. flex 布局下用 `flex: 1` 撑满

Reply:
ok